blivetgui/dialogs/add_dialog.py

```diff
@@ -354,7 +354,8 @@ def _available_add_types(self):
 
         types = []
 
-        if self.selected_parent.type in ("disk", "mdarray"):
+        if self.selected_parent.type == "disk" or (self.selected_parent.type == "mdarray" and
+                                                   self.selected_parent.format.type == "disklabel"):
```

Comment:
The commit message is cryptic and doesn't seem to have too much to do with the actual change.